_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -11,555 +11,245 @@ published: true
-
-```bash
-apx unexport --bin <package>
-```
-
-### Installing Applications in the Fedora Container
-
-- You can install applications in the Fedora container using the following command:-
-
-```bash
-apx install --dnf <packages>
-```
-
-- This command will automatically detect the desktop file entry in the package and add it to the Application menu and Vanilla Control Center.
-
-- Alternatively, you can install the applications by entering the Fedora container using the following commands:-
-
-```bash
-apx enter --dnf
-sudo dnf install <packages>
-exit
-apx export --dnf <packages>
-```
-
-- You can export the installed binary to use it with the host directly using the following command:-
-
-```bash
-apx export --dnf --bin <package>
-```
-
-### Uninstalling Applications from the Fedora Container
-
-- You can uninstall applications from the Fedora container using the following command:-
-
-```bash
-apx remove --dnf <package>
-```
-
-- This command will automatically detect and remove the desktop file entry.
-
-- Alternatively, you can uninstall the applications by entering the Fedora container using the following commands:-
-
-```bash
-apx enter --dnf
-sudo dnf remove <packages>
-exit
-apx unexport --dnf <packages>
-```
-
-- You can unexport the exported binary from the host using the following command:-
-
-```bash
-apx unexport --dnf --bin <package>
-```
-
-### Installing Applications in the Arch Linux Container
-
-- You can install applications in the Arch Linux container using the following command:-
-
-```bash
-apx install --aur <packages>
-```
-
-- This command will automatically detect the desktop file entry in the package and add it to the Application menu and Vanilla Control Center.
-
-- Alternatively, you can install the applications by entering the Arch Linux container using the following commands:-
-
-```bash
-apx enter --aur
-sudo pacman -Syu <packages>
-exit
-apx export --aur <packages>
-```
-
-- You can export the installed binary to use it with the host directly using the following command:-
-
-```bash
-apx export --aur --bin <package>
-```
-
-### Uninstalling Applications from the Arch Linux Container
-
-- You can uninstall applications from the Arch Linux container using the following command:-
-
-```bash
-apx remove --aur <package>
-```
-
-- This command will automatically detect and remove the desktop file entry.
-
-- Alternatively, you can uninstall the applications by entering the Arch container using the following commands:-
-
-```bash
-apx enter --aur
-sudo pacman -Rs <packages>
-exit
-apx unexport --aur <packages>
-```
-
-- You can unexport the exported binary from the host using the following command:-
-
-```bash
-apx unexport --aur --bin <package>
-```
+### Updating Applications
 
-### Installing Applications in the Alpine Linux Container
+Updates are visible in the `Updates` panel in GNOME Software. In Vanilla OS, Updates are available for Flatpaks, Snaps and System Packages through it.
```

Comment:
Not entirely sure how to phrase this differently, but the `through it` part seems off